tests/components/tuya/test_select.py

```diff
@@ -53,3 +58,62 @@ async def test_platform_setup_no_discovery(
     assert not er.async_entries_for_config_entry(
         entity_registry, mock_config_entry.entry_id
     )
+
+
+@pytest.mark.parametrize(
+    "mock_device_code",
+    ["cl_am43_corded_motor_zigbee_cover"],
+)
+async def test_select_option(
+    hass: HomeAssistant,
+    mock_manager: ManagerCompat,
+    mock_config_entry: MockConfigEntry,
+    mock_device: CustomerDevice,
+) -> None:
+    """Test fan mode with windspeed."""
+    entity_id = "select.kitchen_blinds_motor_mode"
+    await initialize_entry(hass, mock_manager, mock_config_entry, mock_device)
+
+    state = hass.states.get(entity_id)
+    assert state is not None, f"{entity_id} does not exist"
+    await hass.services.async_call(
+        SELECT_DOMAIN,
+        SERVICE_SELECT_OPTION,
+        {
+            "entity_id": entity_id,
+            "option": "forward",
```

Comment:
Would be nice if we can change to constants at some point, but not a blocker for this PR